contrib/gorilla/websocket/websocket.go

```diff
@@ -0,0 +1,143 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+// Package websocket provides tracing functions for tracing the gorilla/websocket
+// package (https://github.com/gorilla/websocket).
+package websocket // import "gopkg.in/DataDog/dd-trace-go.v1/contrib/gorilla/websocket"
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/gorilla/websocket"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
```

Comment:
We should create import groups (separated by an empty line).
1. Standard lib
2. Local imports
3. Other imports

Reply:
Good to know, let me add this rule in your guide.

Reply:
added https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines#imports